apps/web-client/src/lib/stores/proto/table_content.ts

```diff
@@ -70,6 +62,8 @@ const mergeBookData = (stock: Iterable<VolumeStockClient>) => (bookData: Iterabl
 		.map(([s, b = {} as BookEntry]) => ({ ...s, ...b }))
 		.array();
 
+//function left if needed for future use
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Yeah, let's not do that: we want to keep linting and TS as strict as possible (unless it's really blocking, annoying and would need more engagement to fix, then we're willing to provide at given moment).
I this case, however, If we ever need the fn, we can either check git history, or, merely, re-create it.